native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mssql-native/resource-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "resources": {
+    "includes": [
+      {
+        "pattern": "\\Qorg/slf4j/impl/StaticLoggerBinder.class\\E"
+      }
+    ]
+  },
+  "bundles": [
+    {
+      "name": "com.microsoft.sqlserver.jdbc.SQLServerResource",
```

Comment:
Is this needed to run the samples or does it used in tests as an external jar file?

Reply:
This is needed to run the samples. Otherwise, getting this error:

```
error: Error in SQL connector configuration: null Caused by :Can't find bundle for base name com.microsoft.sqlserver.jdbc.SQLServerResource, locale en_LK```

Reply:
I see, then lets have that 👍

Reply:
Can we try to add the following locales here and removing this `-H:+AddAllCharsets`?
```json
{
"bundles": [
     {
       "name": "com.microsoft.sqlserver.jdbc.SQLServerResource",
       "locales": [
         "",
         "en-LK"
       ],
       "classNames": [
         "com.microsoft.sqlserver.jdbc.SQLServerResource"
       ]
     }
   ]
}
```

Reply:
When removing this, getting this error:
```
error: Error in SQL connector configuration: Failed to initialize pool: SQL Server collation SQL_Latin1_General_CP1_CI_AS is not supported by this driver. ClientConnectionId:ed919f5d-10c8-4353-973a-4e40afbb2c5c Caused by :SQL Server collation SQL_Latin1_General_CP1_CI_AS is not supported by this driver. ClientConnectionId:ed919f5d-10c8-4353-973a-4e40afbb2c5c Caused by :SQL Server collation SQL_Latin1_General_CP1_CI_AS is not supported by this driver. Caused by :Codepage Cp1252 is not supported by the Java environment.
```